meson.build

```diff
@@ -105,6 +105,7 @@ schemas = [
     'leak_detector.json',
     'firmware.json',
     'supported_configuration.json',
+    'spdm.json',
```

Comment:
may be call this as `spdm_responder.json` to be explicit and not create any confusion ?